.github/workflows/test.yml

```diff
@@ -40,11 +40,11 @@ jobs:
       CONTAINERD_VERSION: "${{ matrix.containerd }}"
       ARCH: "${{ matrix.arch }}"
     steps:
-      - uses: actions/checkout@v4.2.2
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Linter doesn't like the number of the whitespaces
```
43:73 [comments] too few spaces before comment
```

Reply:
Damn.
These things are so picky :D

Reply:
Should be fixed. yamllinter is happy locally.